typescript/web/src/components/invitation-manager/invitation-container.tsx

```diff
@@ -0,0 +1,56 @@
+import { Center, chakra, useColorModeValue as mode } from "@chakra-ui/react";
+import { PropsWithChildren } from "react";
+
+type InvitationContainerProps = {
+  header: string;
+  details: string;
+};
+
+type CompleteProps = PropsWithChildren<InvitationContainerProps>;
+
+export const InvitationContainer = ({
+  header,
+  details,
+  children,
+}: CompleteProps) => {
+  return (
+    <Center h="full">
+      <chakra.div
```

Comment:
While you're at it, can we switch to standard Chakra components?

Reply:
I can but what's wrong with that ?

Reply:
It add extra boilerplate code and complexifies what should have just been an imbrication of VStack, Header and HStack without fixed fontSize, fontWeight, backgrounds, some of the paddings, etc.

Reply:
It's not critical but while there is a few dev bandwidth while waiting for SW to be finished, I think that we can take a few time to clean this up 🧹